packages/fscomponents/src/components/ReviewItem.tsx

```diff
@@ -97,39 +97,46 @@ export class ReviewItem extends Component<ReviewItemProps, ReviewItemState> {
   }
 
   renderSyndicatedIndicator = (): JSX.Element | undefined => {
-    if (this.props.syndicationSource && this.props.syndicationSource.LogoImageUrl) {
+    if (
+      !this.props.renderSyndicatedIndicator &&
+      this.props.syndicationSource &&
+      this.props.syndicationSource.LogoImageUrl
+    ) {
 
       Image.getSize(
         this.props.syndicationSource.LogoImageUrl,
         this.getImageSizeSuccess,
         () => null
       );
 
-      return this.props.renderSyndicatedIndicator ?
-        this.props.renderSyndicatedIndicator(this.props.syndicationSource) :
-        (
-          <View
-            style={[
-              S.row, { flexDirection: 'row', flexWrap: 'wrap', paddingBottom: 20 },
-              this.props.rowStyle
-            ]}
-          >
-            <Image
-              style={{
-                height: this.state.syndicatedImageHeight,
-                width: this.state.syndicatedImageWidth,
-                marginRight: 6
-              }}
-              source={{uri: this.props.syndicationSource.LogoImageUrl}}
-              accessibilityLabel={`${this.props.syndicationSource.Name} logo`}
-            />
-            <Text style={[S.syndicatedLabel]}>
-              {FSI18n.string(componentTranslationKeys.syndicatedLabel, {
-                site: this.props.syndicationSource.Name
-              })}
-            </Text>
-          </View>
-        );
+      return (
+        <View
+          style={[
+            S.row, { flexDirection: 'row', flexWrap: 'wrap', paddingBottom: 20 },
+            this.props.rowStyle
+          ]}
+        >
+          <Image
+            style={{
+              height: this.state.syndicatedImageHeight,
+              width: this.state.syndicatedImageWidth,
+              marginRight: 6
+            }}
+            source={{uri: this.props.syndicationSource.LogoImageUrl}}
+            accessibilityLabel={`${this.props.syndicationSource.Name} logo`}
+          />
+          <Text style={[S.syndicatedLabel]}>
+            {FSI18n.string(componentTranslationKeys.syndicatedLabel, {
+              site: this.props.syndicationSource.Name
+            })}
+          </Text>
+        </View>
+      );
+    } else if (
+      this.props.renderSyndicatedIndicator &&
+      this.props.syndicationSource &&
+      this.props.syndicationSource.LogoImageUrl) {
+      return this.props.renderSyndicatedIndicator(this.props.syndicationSource);
```

Comment:
Making the if checks like this should make it a little cleaner and require fewer checks:
```
if (this.props.syndicationSource && this.props.syndicationSource.LogoImageUrl) {
  if (this.props.renderSyndicatedIndicator) {
    return this.props.renderSyndicatedIndicator(this.props.syndicationSource);
  } else {
    return (big block of jsx);
  }
} else {
  return;
}
```

You might even be able to cut the last return.